rllib/models/base_model.py

```diff
@@ -0,0 +1,244 @@
+"""Model base classes.
+
+RLlib models all inherit from the recurrent base class, which can be chained together
+with other models.
+
+The models input and output TensorDicts. Which keys the models read/write to
+and the desired tensor shapes must be defined in input_spec, output_spec,
+prev_state_spec, and next_state_spec.
+
+The `unroll` function gets the model inputs and previous recurrent state, and outputs
+the model outputs and next recurrent state. Note all ins/outs must match the specs.
+Users should override `_unroll` rather than `unroll`.
+
+`initial_state` returns the "next" state for the first recurrent iteration. Again,
+users should override `_initial_state` instead.
+
+For non-recurrent models, users may instead override `_forward` which does not
+make use of recurrent states.
+"""
+
+
+import abc
+from typing import Optional, Tuple, Any
+
+
+# TODO: Remove once TensorDict is in master
+class TensorDict:
```

Comment:
stick these placeholder classes out of this file and put them next to `base_model.py` under `temp_spec_classes.py`

Reply:
On top of that module explain why this is a temporary module.